float32[6] poseVector
